### PR TITLE
app-crypt/libscrypt: Add static-libs USE flag and multilib support

### PR DESCRIPTION
**Pull request by Xiami2012**
Also:
Update to EAPI=7
Avoid redefinition of _FORTIFY_SOURCE

Package-Manager: Portage-2.3.24, Repoman-2.3.6
Signed-off-by: Xiami <i@f2light.com>

### CODE REVIEW

File: app-crypt/libscrypt/libscrypt-1.21-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs multilib-minimal
```
Comment (hydrapolic) [inline app-crypt/libscrypt/libscrypt-1.21-r1.ebuild]: Please sort.

Comment (Xiami2012) [reply app-crypt/libscrypt/libscrypt-1.21-r1.ebuild]: If multiple eclass EXPORT_FUNCTIONS the same phase function(e.g. src_configure), the last inherited one wins.
So I put mulitlib-minimal the last to remind follow-up maintainers to be caution when involving new eclasses.

And, https://github.com/gentoo/gentoo/blob/master/eclass/multilib-minimal.eclass#L16 says it should be the last in inherit order.

But, it's okay to sort, as toolchain-funcs has no exports in this case. Please confirm. 😃

Comment (hydrapolic) [reply app-crypt/libscrypt/libscrypt-1.21-r1.ebuild]: Then ok, leave as is, i haven't read multilib-minimal :)

**Comment by Alessandro-Barbieri** [comment]
Do you really need the static-libs useflag?
I have two readings for you
https://archives.gentoo.org/gentoo-dev/message/2dada80c2b9c85b0e83e6328428bf8ab
https://flameeyes.blog/2011/08/29/useless-flag-static-libs/
If not please remove the static-libs use flag

**Comment by ajakk** [comment]
Changes have been requested for approaching two years now, I guess no one is there anymore.

**Pull request closed without merging**